packages/server/assets/serverinvites/typedefs/serverInvites.graphql

```diff
@@ -100,7 +100,7 @@ extend type Query {
   """
   Receive metadata about an invite by the invite token
   """
-  serverInviteByToken(token: String!): ServerInvite
+  serverInviteByToken(token: String): ServerInvite
```

Comment:
To allow for easier querying (both server & workspaceInvite w/ same query). Resolver just returns null if token empty.